src/tools/data/datagen/wave.C

```diff
@@ -53,9 +53,9 @@
 
 #include "silo.h"
 
-// supress the following since silo uses char * in its API
+// suppress the following since silo uses char * in its API
```

Comment:
After seeing a ton of instances of this bit of code in VisIt's test generation sources...I am wondering if either a) it should really be `silo.h` or if there is an include file we should use there that codifies this logic once for all the cases it is needed?

Reply:
@markcmiller86 I think it's fine here. If you put it in `silo.h`, that would squash this warning for any code that includes `silo.h`.  We accept the silo api isn't const friendly -- but that would also skip the warnings for calls unrelated to silo, w/o folks knowing. Could cause an issue in downstream simulation codes, etc